staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -257,7 +270,8 @@ func (o *DebugOptions) Complete(restClientGetter genericclioptions.RESTClientGet
 	}
 
 	if o.Applier == nil {
-		applier, err := NewProfileApplier(o.Profile)
+		keeps := NewKeepFields().WithKeepLabels(o.KeepLabels).WithKeepAnnotations(o.KeepAnnotations).WithKeepLiveness(o.KeepLiveness).WithKeepReadiness(o.KeepReadiness).WithKeepStartup(o.KeepStartup).WithKeepInitContainers(o.KeepInitContainers)
```

Comment:
Instead of all these functions, what about;
```suggestion
		keeps := &KeepFields{
		Labels: o.KeepLabels,
		Annotations: o.KeepAnnotations
		...
		}
```
I know I was the suggester of builder pattern but unfortunately it seems redundant now as `ProfileApplier` is actually an interface and we don't want to change it's signature.